tests/language-service/languageservicerunner.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference path="../../built/pxtcompiler.d.ts"/>
 
```

Comment:
All changes in this file are new. I added support for testing snippets (toolbox or auto-complete).